TODO

```diff
@@ -9,4 +9,7 @@ Infra TODO:
 
 
 Src todo:
-- implement the actual algorithms in sanitizer.js
\ No newline at end of file
+- implement the actual algorithms in sanitizer.js
+
+eslint todo:
+- might be worth trying https://github.com/amilajack/eslint-plugin-compat to help with polyfill development
```

Comment:
:+1: great idea

 I created github issue #6 for that, let's move to those and slowly but steadily remove this file :)

Reply:
> 👍 great idea

Thanks! :)

> I created github issue mozilla#6 for that, let's move to those and slowly but steadily remove this file :)

I made a separate [PR](https://github.com/mozilla/sanitizer-polyfill/pull/8) for https://github.com/mozilla/sanitizer-polyfill/issues/6. I'll remove this TODO from the file and will try to add future TODOs as GitHub issues instead